### DISCUSSION TITLE
Hiding and reordering devices

### DISCUSSION THREAD

**Discussion by Jack-Whattam** [nokyan/resources#213] [General]
I haven't created a issue/request as I figured I may have missed something or this might be something specific with my installation choice (installed from AUR on EndeavourOS). Would be nice if the devices shown in the sidebar could be reordered or hidden. I do not use an ethernet connection for example so it would be nice to hide this and move the Wi-Fi connection info further up. Otherwise, great app, much better than anythign else I have tried and it properly recognised both AMD and Nvidia GPU's immediately!

**Comment by nokyan** [comment]
Hi,
sorry for responding this late, I didn't have notifications turned on for discussions.

I'm sorry but I think I will implement hiding and reordering individual devices in the near future because that's quite tricky for something that is as unpredictable and "inconstant" as network interfaces and drives, especially with the way recognizing new devices and removing newly disconnected devices works right now.
